Source/SpatialGDK/Private/Interop/Connection/SpatialConnection.cpp

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialConnection.h"
+
+#include "Async/Async.h"
+
+void USpatialConnection::FinishDestroy()
```

Comment:
We've already got a USpatialNetConnection, this seems a little too closely named.  SpatialBridge?

Reply:
I think `Bridge` is fairly misleading, since bridge means something else. I personally think `Connection` is ok because `NetConnection` is in a different folder and used in a completely different context.

Reply:
Perhaps `Bridge` isn't the best, but having a SpatialConnection and a SpatialNetConnection is a bad idea. We need to rename one.

Reply:
Agree with Mike, this at least needs to be `USpatialWorkerConnection`.